web/src/components/atoms/DownloadButton/index.tsx

```diff
@@ -19,7 +18,20 @@ const DownloadButton: React.FC<DownloadButtonProps> = ({
   ...props
 }) => {
   const handleDownload: MouseEventHandler<HTMLElement> | undefined = () => {
-    saveAs(url, filename);
+    fetch(url, {
```

Comment:
Can you explain why you stopped using file-saver and are kinda doing it manually now?

Reply:
When I try to download an image with file-saver, it gets opened in a new tab rather than downloading.

Reply:
In reearth-web I use this [js-file-download package](https://github.com/eligrey/FileSaver.js) which works well if all you need is to download the specified file to the user's computer!

Reply:
[js-file-download package](https://github.com/eligrey/FileSaver.js) is the same as file-saver.
it worked on my computer in the past, but recently for some reason it stopped working and started to open files in new tabs instead of downloading them.
if you think that file-saver is reliable, I will revert the changes that I have done to the DownloadButton component and keep using file-saver instead.

Reply:
I used js-file-downoad instead of file-saver